python/restamp_bag.py

```diff
@@ -39,14 +39,14 @@ def main(argv):
             outputfile = arg
 
     # print console header
-    print ""
-    print "restamp_bag"
-    print ""
-    print 'input file:  ', inputfile
-    print 'output file: ', outputfile
-    print ""
-    print "starting restamping (may take a while)"
-    print ""
+    print("")
```

Comment:
You can also do the following

```Python
print("""
a
b
c
d\n
f
g""")
```
which yields the following output:
```
a
b
c
d

f
g
```

or this, which looks a bit nicer in my opinion, and is more suitable for this case:
```Python
print("a"
      "b"
      "c"
      "d\n"
      "f"
      "g")
```
which results in:
```
abcd
fg
```
especially the `print("")` looks a bit ugly in my opinion.